### ISSUE TITLE
🌐 Website Issue | Hosting providers have no warning (unlike VPN/DNS)

### ISSUE THREAD

**Issue by Mikaela** [privacytoolsIO/privacytools.io#1341]
The VPN and DNS providers warn that we cannot know what they are doing and are trusting their word on it. The hosting providers have no such a warning and the page only says:

> Web Hosting
> Find a web hosting provider that won't track your visitors or give into government data requests.

**Comment by blacklight447-ptio** [comment]
I agree, maybe even provide a notice that even when a service makes its server software open source, it has no way of proving that it really runs THAT software.
